### ISSUE TITLE
DataGrid inline editor not showing controls when editing

### ISSUE THREAD

**Issue by mrpmorris** [Megabit/Blazorise#4132]
```
    <DataGrid Data=@ServiceTypes Sortable Striped Responsive Bordered Editable Hoverable
              TItem="ServiceTypeListItemModel"
              EditMode=@DataGridEditMode.Inline
              RowInserting=@CreateAsync>
        <DataGridColumns>
            <DataGridCommandColumn>
            </DataGridCommandColumn>
            <DataGridColumn Caption="Name" Field=@nameof(ServiceTypeListItemModel.Name)>
                <EditTemplate>
                    xxx <InputText @bind-Value=context.Item!.Name /> xxx
                </EditTemplate>
            </DataGridColumn>
        </DataGridColumns>
    </DataGrid>
```

When I click the `New` or `Edit` links, I do not see the content of the `EditTemplate` but only the same content as before.

**Comment by stsrki** [comment]
What can be seen when you inspect in dev tools?

**Comment by mrpmorris** [comment]
Before
```
<tr class="table-row-selectable"><!--!--><!--!--><td><!--!--><button id="0HMKVSHSO7M0A" type="button" class="btn btn-link" _bl_19="">Edit</button><!--!--><button id="0HMKVSHSO7M0B" type="button" class="btn btn-link" _bl_20="">Delete</button></td><!--!--><td>16+ Accommodation &amp; Support</td></tr>
```

After
```
<tr><!--!--><!--!--><td><!--!--><button id="0HMKVSHSO7M0C" type="submit" class="btn btn-link" _bl_21="">Save</button><!--!--><button id="0HMKVSHSO7M0D" type="button" class="btn btn-link" _bl_22="">Cancel</button></td><!--!--><td>16+ Accommodation &amp; Support</td></tr>
```

**Comment by stsrki** [comment]
I think you're missing Editable on DataGridColumn

```
<DataGridColumn Caption="Name" Field=@nameof(ServiceTypeListItemModel.Name) Editable>
```

**Comment by mrpmorris** [comment]
Ah yes, that's it, thank you!

The documentation only says to add it to the DataGrid itself, and the `Editable` attribute on the line is the last attribute and by default isn't visible on my screen.

May I suggest you alter the docs
> The grid can perform some basic CRUD operations on the supplied Data collection. To enable editing on data-grid just set the Editable attribute to true.

to

> The grid can perform some basic CRUD operations on the supplied Data collection. To enable editing on data-grid, set the Editable attribute to true on the DataGrid, and then set Editable to true on each column you wish to be editable.

Also, move the `Editable` on the DataGridColumn all the way to the start of the line.  It looks out of place, but that's what will make it noticeable!

Thanks for your help!

**Comment by stsrki** [comment]
> > The grid can perform some basic CRUD operations on the supplied Data collection. To enable editing on data-grid, set the Editable attribute to true on the DataGrid, and then set Editable to true on each column you wish to be editable.

Done, Thanks for the recommendation. It will be available today along with v1.1.
